src/collective/cover/browser/compose.py

```diff
@@ -50,10 +50,16 @@ def render(self):
             path = '{0}/{1}'.format(self.tile_type, self.tile_id)
             tile = self.context.restrictedTraverse(path)
             tile.populate_with_object(obj)
+            # reinstantiate tile to update rendering with new data
+            tile = self.context.restrictedTraverse(path)
             return tile()
 
     def __call__(self):
         self.setup()
+        # avoid caching the response on intermediate proxies
+        # as this is a GET request, a misconfiguration could
+        # result on outdated content being rendered
+        self.request.response.setHeader('Cache-Control', 'no-cache')
```

Comment:
@fredvd I was checking your previous commits about this and I was curious about the inclusion of `must-revalidate` in the `Cache-Control` header; AFAIK, that is only needed if the content is cached.

comments?